test/e2e/networking.go

```diff
@@ -79,7 +77,7 @@ var _ = Describe("Networking", func() {
 	//This namespace is modified throughout the course of the test.
 	var namespace *api.Namespace
 	var svcname = "nettest"
-	var c *client.Client = nil
+	var cli *client.Client = nil
```

Comment:
Can you keep the variable named 'c' so it's consistent with the rest of the e2e tests?